man/rgblink.5

```diff
@@ -1,53 +1,37 @@
 .\" SPDX-License-Identifier: MIT
 .\"
-.Dd March 28, 2021
+.Dd December 22, 2023
```

Comment:
We should update the other man pages' dates, they haven't been consistently done with content updates. (Should they be to "the last time each file was modified" or "all the day of the mass-update commit"?)

Reply:
Preferably the former, yeah.

Reply:
Filed as #1278.